Cryptomator/VaultList/VaultListViewController.swift

```diff
@@ -56,6 +56,15 @@ class VaultListViewController: ListViewController<VaultCellViewModel> {
 		}
 	}
 
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
+		if CryptomatorUserDefaults.shared.showOnboardingAtStartup {
+			coordinator?.showOnboarding()
+		} else if UserDefaultsFullVersionChecker.shared.hasExpiredTrial, !CryptomatorUserDefaults.shared.showedTrialExpiredAtStartup {
```

Comment:
```suggestion
		} else if UserDefaultsFullVersionChecker.shared.hasExpiredTrial, CryptomatorUserDefaults.shared.showTrialExpiredAtStartup {
```

Reply:
I'm not sure if we should rename it to `showTrialExpiredAtStartup` because it has a different semantic than `showOnboardingAtStartup`.
As `showedTrialExpiredAtStartup` just indicates if we already showed the "Trial Expired"-screen to the user.
However, we also need to check (via `hasExpiredTrial`) if we encountered an expired trial.
Therefore, in my opinion `showTrialExpiredAtStartup` does not really indicate if we should show the "Trial Expired"-screen.

Maybe we refactor this into a function `showTrialExpired` inside of the `VaultListViewController` to make it a bit more clear?